modules/data.atmosphere/R/met2cf.ERA5.R

```diff
@@ -21,28 +21,36 @@
 #' }
 met2cf.ERA5 <- function(lat,
                         long,
-                        years,
+                        start_date,
+                        end_date,
                         sitename,
-                        data.folder,
                         outfolder,
+                        dbparms,
```

Comment:
As noted previously, met functions shouldn't access the database. They shouldn't need to do so, and doing so breaks the ability to run them remote.